app/queries/get_duplicate_candidate_matches.rb

```diff
@@ -1,25 +1,25 @@
 class GetDuplicateCandidateMatches < ApplicationRecord
   def self.call
     ActiveRecord::Base.connection.exec_query(
-      "SELECT a.candidate_id, a.first_name, a.last_name, a.postcode, a.date_of_birth, email_address
-    FROM application_forms a
+      "SELECT application_details.candidate_id, application_details.first_name, application_details.last_name, TRIM(UPPER(application_details.postcode)), application_details.date_of_birth, email_address
```

Comment:
You can give the generic `btrim` field a name by following the column declaration with an alias:

```sql
TRIM(UPPER(application_details.postcode)) postcode
```